alf/algorithms/actor_critic_loss.py

```diff
@@ -81,13 +106,23 @@ def __init__(self,
         self._use_gae = use_gae
         self._lambda = td_lambda
         self._use_td_lambda_return = use_td_lambda_return
+        if normalize_scalar_advantages:
```

Comment:
For me, I feel it's more natural that `normalize_advantages=True` is a pre-condition for using `normalize_scalar_advantages` which basically determines how we are going to normalize the advantage: either for individual reward dims or for the combined scalar.

Reply:
I feel it's cumbersome to need to specify both normalize_advantages=True normalize_scalar_advantages=True